cmd/mir-validator/init.go

```diff
@@ -52,20 +52,26 @@ var initCmd = &cli.Command{
 		}
 
 		// TODO: Pass validator set for initialization
-		mp := path.Join(cctx.String("repo"), MembershipCfgPath)
+		membershipFile := path.Join(cctx.String("repo"), MembershipPath)
 		if cctx.String("membership") != "" {
-			validators, err := mir.GetValidatorsFromFile(cctx.String("membership"))
+			validators, err := validator.NewValidatorSetFromFile(cctx.String("membership"))
 			if err != nil {
 				return fmt.Errorf("error importing membership config specified: %s", err)
 			}
 			// persist validator config in the right path.
-			if err := mir.ValidatorsToCfg(validators, mp); err != nil {
+			if err := validators.StoreToFile(membershipFile); err != nil {
 				return fmt.Errorf("error exporting membership config: %s", err)
 			}
 		} else {
+<<<<<<< HEAD
```

Comment:
You seem to have introduced an unresolved conflict here :shrug:

Reply:
Interesting! I do not have this line in my branch.